faqs/galaxy/datasets_custom_genomes.md

```diff
@@ -0,0 +1,23 @@
+---
+title: What is a "Custom Reference Genome" ?
```

Comment:
the content of the FAQ is addressing the definition (what) and the how, maybe we can rephrase the title?